components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -331,14 +331,16 @@ def render_thumbnail(request, iid, w=None, h=None, conn=None, _defcb=None,
     """
     server_id = request.session['connector'].server_id
     direct = True
+    defaultSize = conn.getDefaultThumbnailSize()
```

Comment:
It may be easier to just add it to omeroweb.settings instead, unless there is a need to share it between clients